### PR TITLE
SSF content

### PR DESCRIPTION
**Pull request by emileighoutlaw**
This adds the OGC approved SSF content to the Registration Options page. 

Prepping content for release this week.

cc @noahmanger @jenniferthibault

### CODE REVIEW

File: fec/home/templates/home/landing_page.html [hunk 1]
```diff
@@ -78,8 +78,19 @@ <h5 class="label">Campaign guide</h5>
       </div>
       <div id="corporations-and-labor" class="option">
         <div class="option__content">
-          <h3>Coming soon: Corporations and Labor Organizations</h3>
-          <p>This is the FEC’s beta site, which means it’s still being developed. Learn more by reading the campaign guide, and check back soon for more information about corporations and labor organizations.</p>
+          <h3>Corporations and labor organizations</h3>
+          <p>Corporations and labor organizations can't contribute to federal candidates, but they can establish and administer a special kind of political committee: the separate segregated fund (SSF).</p>
+          <p>An SSF can raise funds to make contributions to candidates — including expenditures that are coordinated with candidates — within the applicable contribution limits.</p>
```
Comment (jenniferthibault) [inline fec/home/templates/home/landing_page.html]: @emileighoutlaw quick check: is it _**an**_ SSF or _**a**_ SSF.
My ears say An, but reading it my brain says S is not a vowel, so A.

Comment (emileighoutlaw) [inline fec/home/templates/home/landing_page.html]: Excellent question! The ear test is correct! If it's an acronym and vowel sound,  *an* is correct. This is one of the few times where I feel like grammar makes common sense! :dancer:

**Comment by jenniferthibault** [comment]
Awesome, I removed one small extra space earlier, so looks good to me! Merging.

**Comment by jenniferthibault** [comment]
Once the essentials pages are ready, we'll need to come back and put the link in on this headline.

**Comment by emileighoutlaw** [comment]
Great! Thanks for the note reminder :)

**Pull request merged**